Updating api doc link

shreelakshmijoshi opened the pull request:
* **Please check if the PR fulfills these requirements** :clipboard:
- [x] The commit message follows our guidelines : [link](https://github.com/datakaveri/iudx-acl-apd/blob/main/CONTRIBUTING.md)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] A new branch is created for the change(s) and the changes are not committed on the main or master branch
- [x] No credentials or secrets are committed

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, refactor, chore, test) :information_desk_person: :scroll:

  _docs update_, _refactor_ <br>
 - Updated the README.md to have the Redocly URL of the OpenAPI specs
 -  Refactored PR template
 - Added CDPG .svg icon in Open API specs 


* **Checklist** :page_with_curl:
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added reviewers to check the proposed changes

abhi4578 commented:
> * **Please check if the PR fulfills these requirements** 📋
> 
> 
>     * [x]  The commit message follows our guidelines : [link](https://github.com/datakaveri/iudx-acl-apd/blob/main/CONTRIBUTING.md)
> 
>     * [x]  Docs have been added/updated (for bug fixes/features)
> 
>     * [x]  A new branch is created for the change(s) and the changes are not committed on the main or master branch
> 
>     * [x]  No credentials or secrets are committed
> 
> 
>     * **What kind of change does this PR introduce?** (Bug fix, feature, docs update, refactor, chore, test) 💁 📜
>       _docs update_, _refactor_
> 
> 
>     * Updated the README.md to have the Redocly URL of the OpenAPI specs
> 
>     * Refactored PR template
> 
> 
>     * **Checklist** 📃
> 
> 
>     * [x]  I have made corresponding changes to the documentation
> 
>     * [x]  My changes generate no new warnings
> 
>     * [x]  I have added reviewers to check the proposed changes

Hey @shreelakshmijoshi , can we have followings point in checklist also 
 - If config options changed, is it added in git (./example-config/config.json), notify  devops team with mention ``` @datakaveri/devops ``` in comment,  to add changes in CI pipeline and for deployment 
 - If any backend components ( postgres (flyway files), immudb, redis, rabbitmq, elasticsearch) configs changed, notify  devops team with mention ``` @datakaveri/devops ``` in comment, for necessary changes in [iudx-deployment](https://github.com/datakaveri/iudx-deployment) repository and deployments in the environments.
- Explicitly mention as comment if this PR requires any cat items  (commit the cat items needed for server in src/test/resources/cat-items?, have a brief readme also) to be created on or depends on for testing of the functionality. 

  \+cc @karun-singh , anything else we would need?

shreelakshmijoshi commented:
> > * **Please check if the PR fulfills these requirements** 📋
> >   
> >   * [x]   The commit message follows our guidelines : [link](https://github.com/datakaveri/iudx-acl-apd/blob/main/CONTRIBUTING.md)
> >   * [x]   Docs have been added/updated (for bug fixes/features)
> >   * [x]   A new branch is created for the change(s) and the changes are not committed on the main or master branch
> >   * [x]   No credentials or secrets are committed
> >   * **What kind of change does this PR introduce?** (Bug fix, feature, docs update, refactor, chore, test) 💁 📜
> >     _docs update_, _refactor_
> >   * Updated the README.md to have the Redocly URL of the OpenAPI specs
> >   * Refactored PR template
> >   * **Checklist** 📃
> >   * [x]   I have made corresponding changes to the documentation
> >   * [x]   My changes generate no new warnings
> >   * [x]   I have added reviewers to check the proposed changes
> 
> Hey @shreelakshmijoshi , can we have followings point in checklist also
> 
> * If config options changed, is it added in git (./example-config/config.json), notify  devops team with mention `@datakaveri/devops` in comment,  to add changes in CI pipeline and for deployment
> * If any backend components ( postgres (flyway files), immudb, redis, rabbitmq, elasticsearch) configs changed, notify  devops team with mention `@datakaveri/devops` in comment, for necessary changes in [iudx-deployment](https://github.com/datakaveri/iudx-deployment) repository and deployments in the environments.
> * Explicitly mention as comment if this PR requires any cat items  (commit the cat items needed for server in src/test/resources/cat-items?, have a brief readme also) to be created on or depends on for testing of the functionality.
>   +cc @karun-singh , anything else we would need?

Hello Abhilash,
Thanks for suggesting the changes. I have added the changes here : [Update PR Template](https://github.com/datakaveri/iudx-acl-apd/pull/163/commits/ee4854c7844c666b94b81a6ff1d66c979b70414a), [Update markdown file](https://github.com/datakaveri/iudx-acl-apd/pull/163/commits/ec22f591d464e9160fb413615024229bf8f2912d)

shreelakshmijoshi commented:
retest this please
